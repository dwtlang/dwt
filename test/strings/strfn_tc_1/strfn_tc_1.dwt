fun foo() {
}

print str(foo)
print str(true)
print str(false)
print str(nil)

var i := 123.456
var j := 654.321

print str(i) + str(j)

