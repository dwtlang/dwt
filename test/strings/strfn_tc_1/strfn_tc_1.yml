---
description:      "test built-in str() casting function"
name:             strfn_tc_1
src:              strfn_tc_1.dwt
out:              strfn_tc_1.out
err:              strfn_tc_1.err
exitcode:         0
loop:             1
skip:             false
