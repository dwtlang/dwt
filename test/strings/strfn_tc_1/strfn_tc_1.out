<fun ::foo>
true
false
<nil>
123.456654.321
